ci-bootstrap/README.md

```diff
@@ -0,0 +1,7 @@
+Resources for setting up roles in a new AWS account
```

Comment:
Need to update README with instructions for deploying via pipeline etc

Reply:
Created a readme file for the CI